### PR TITLE
feat!: DVC-8020 rename DVC to DevCycle on public interfaces

### PR DESCRIPTION
**Pull request by jonathannorris**

### CODE REVIEW

**Review by suthar26 — APPROVED** [review]
All changes look safe and properly deprecated the old references

**Comment by jonathannorris** [comment]
@jsalaber let me know if you want to review this.

File: DevCycleTests/Models/DevCycleClientTests.swift [hunk 1]
```diff
@@ -58,8 +58,18 @@ class DVCClientTest: XCTestCase {
         client.close(callback: nil)
     }
     
+    func testDepracatedDVCClientWorks() {
```
Comment (jsalaber) [inline DevCycleTests/Models/DevCycleClientTests.swift:61]: `Deprecated`

**Review by jsalaber — APPROVED** [review]
small typo

**Pull request merged**